Formula/rabbitmq.rb

```diff
@@ -1,8 +1,8 @@
 class Rabbitmq < Formula
   desc "Messaging broker"
   homepage "https://www.rabbitmq.com"
-  url "https://dl.bintray.com/rabbitmq/binaries/rabbitmq-server-generic-unix-3.7.1.tar.xz"
-  sha256 "66622252372ea70f5cf1eb60fa9421dd933834272823ea269b2435d3ab33bafb"
+  url "https://dl.bintray.com/rabbitmq/all/rabbitmq-server/3.7.2/rabbitmq-server-generic-unix-3.7.2.tar.xz"
```

Comment:
why are you changing the URL format? The previous style should still work, and is easier to maintain since the version number only needs to be changed in one place.

Reply:
That's the new layout of RabbitMQ's Bintray repository. One of the goals of switching to Bintray was to support N versions per repository (this is trivial for binary artifacts but less so Debian and RPMs) and that's the layout our team decided to go with for reasons that are not very interesting to the users.

A single directory for all artifacts we ever produce doesn't seem to be a very bright decision [which I made] in retrospect.

Reply:
One specific example. Say you need to find an artifact from a version that's not the most recent one. Which one is easier to navigate, [the old layout](https://dl.bintray.com/rabbitmq/binaries/) or [the new one](https://dl.bintray.com/rabbitmq/all/rabbitmq-server/)? Machines don't care but humans do. Some of it can be attributed to the way Bintray UI works but that's not something my team can influence easily (or at all).

Previously this formula was using GitHub for downloads. Artifact URLs on GitHub also include the version number twice, e.g. `https://github.com/rabbitmq/rabbitmq-server/releases/download/v3.7.2/rabbitmq-server-generic-unix-3.7.2.tar.xz`.

Reply:
:sob:

Reply:
Since this is Ruby code, a constant can be extracted to eliminate this "duplication". I'll submit a PR for that.

Reply:
That style isn't permitted in Homebrew.